packages/orchestration/src/examples/swapExample.contract.js

```diff
@@ -40,10 +40,10 @@ export const makeNatAmountShape = (brand, min) =>
 /**
  * @param {ZCF} zcf
  * @param {{
- * localchain: ERef<LocalChain>;
- * orchestrationService: ERef<OrchestrationService>;
- * storageNode: ERef<StorageNode>;
- * timerService: ERef<TimerService>;
+ * localchain: Remote<LocalChain>;
+ * orchestrationService: Remote<OrchestrationService>;
+ * storageNode: Remote<StorageNode>;
+ * timerService: Remote<TimerService>;
```

Comment:
That could be an API change from allowing promises to not allowing promises. It looks like `storageNode` is matched against an `M.remotable()` patterns, so promises are not allowed, at least in that case.

Is it worth changing the rest of `privateArgsShape` from `M.any()` to `M.remotable()` likewise to be explicit? It's orthogonal, but maybe worthwhile, while we're in the neighborhood?